src/snowflake/cli/api/project/schemas/project_definition.py

```diff
@@ -52,3 +52,15 @@ def _is_supported_version(cls, version: str) -> str:
 
     def meets_version_requirement(self, required_version: str) -> bool:
         return Version(self.definition_version) >= Version(required_version)
+
+    @model_validator(mode="after")
+    def check_card_number_omitted(self) -> ProjectDefinition:
+        from snowflake.cli.api.console import cli_console
+
+        # Check for env section
+        if self.env and not self.meets_version_requirement("1.1"):
+            cli_console.warning(
+                "The `env` section is supported since definition_version: 1.1 "
+            )
+
+        return self
```

Comment:
Does it work withouth changing `extra` setting here? 😱 https://github.com/snowflakedb/snowflake-cli/blob/6ad65a40af611a372d0429f03dd35fff8571c93c/src/snowflake/cli/api/project/schemas/updatable_model.py#L11